src/containers/Map/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import styles from './styles.sass'
+import L from 'leaflet'
+
+export default class Map extends Component {
+  componentWillReceiveProps() {
```

Comment:
empty methods definitions should be usually highlighted by ESLint (e.g. [airbnb](https://github.com/airbnb/javascript/tree/master/react)) - btw, #1 implies a linter :) We could also configure `webpack` to trigger linter via a [pre-loader](https://github.com/MoOx/eslint-loader)